3.0/vector_tile.proto

```diff
@@ -69,6 +112,12 @@ message Tile {
                 // See https://github.com/mapbox/vector-tile-spec/issues/47
                 optional uint32 extent = 5 [ default = 4096 ];
 
+                repeated string string_values = 7;
+                repeated double double_values = 8 [ packed = true ];
+                repeated float float_values = 9 [ packed = true ];
```

Comment:
Can we try to keep the types ordered consistently throughout the `.proto` file, ie some places have `float` first, then `double`, others in different order.

Reply:
Sounds good to me. I'll make that edit.